### PR TITLE
RCHAIN-4012: deploy - check for minimum phlo price

### PR DESCRIPTION
**Pull request by tgrospic**
## Overview
Before processing deploy check for minimum phlo price. gRPC and CLI client will receive an error message.

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-4012

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala [hunk 1]
```diff
@@ -452,7 +452,7 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
 
     val phloPrice = opt[Long](
       descr = "The price of phlo for this transaction in units dust/phlo. Must be positive integer.",
-      validate = _ > 0,
+      validate = _ >= 0,
```
Comment (tgrospic) [inline node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala:456]: This validation is only for CLI. Zero is validated on executing node so gRPC and CLI will return the same error message.

Comment (nzpr) [reply node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala:456]: @tgrospic it diverges from `descr`, which says `Must be positive integer`. Could you add the same comment in the code to omit confusion in future?

**Review by zsluedem — APPROVED** [review]
LGTM

**Review by nzpr — APPROVED** [review]

**Pull request merged**